### DISCUSSION TITLE
Google前員工重新發明的瀏覽器- Arc

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS3073-C/Python-Community-GS3073-C#600] [General]
Google 前員工 Josh Miller 在2019成立了The Browser Company這個新創公司，目的是「重新發明」瀏覽器，打造更直覺、流暢、靈活的瀏覽器體驗，他認為市面上主流的瀏覽器受限於用戶的慣性、畫面看起來都一樣、有著同樣的UI/UX，用戶需要另一個不同的選擇。

他們所打造的瀏覽器-Arc目前僅有macOS版本，且僅提供部分人員測試，剛好有拿到邀請連結的我就下載來玩玩看了，我覺得最大的不同是可以在一個畫面中同時顯示不同的分頁，這對常常需要同時使用兩個分頁的我來說，省去了許多切換分頁的時間（尤其是使用單螢幕的時候，使用雙螢幕的同學應該影響不大XD），畫面如下:
![Screenshot 2023-01-06 at 01 05 28 (2)](https://user-images.githubusercontent.com/32983347/210839438-56d01559-247d-414b-923d-c2b81a272fd5.png)

不過目前發現的缺點是Arc似乎還沒有開發人員工具(F12)，但這對於一般使用者來說也不怎麼影響使用，整體來說使用體驗是很不錯的，希望他能在瀏覽器這個難以生存的市場中存活下去！

資料來源: https://tw.news.yahoo.com/news/arc-%E4%B8%8D%E5%8F%AA%E6%98%AF-chrome-%E7%9A%84%E6%9B%BF%E4%BB%A3%E5%93%81-%E5%89%8D-101040110.html

**Comment by RogelioKG** [comment]
感覺瀏覽器還是要有一些新人加入競爭，
才會讓整個市場添入一些活水。
90年代是 Netscape 的天下，
00年代是 IE 的獨霸，
10年代是 Chrome 的掌控，
期待下個 10 年的變化。
